core/storage/user/gae_models.py

```diff
@@ -161,6 +161,25 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     # The average of average ratings of all explorations.
     average_ratings = ndb.FloatProperty(indexed=True)
 
+    # A list which stores history of creator stats.
+    # Each item in the list is a Json object keyed by a datetime string and
+    # value as another Json object containing key-value pairs to be stored.
+    # [
+    #  {
+    #   (date_1): {
+    #    "average_ratings": 4.3,
+    #    "total_plays": 40
+    #   }
+    #  },
+    #  {
+    #   (date_2): {
+    #    "average_ratings": 4.1,
+    #    "total_plays": 60
+    #   }
+    #  },
+    # ]
+    weekly_creator_stats_list = ndb.JsonProperty(repeated=True, default=None)
```

Comment:
Don't think you need a default if it's repeated, right? https://cloud.google.com/appengine/docs/python/ndb/entity-property-reference#options